x-pack/plugins/apm/public/components/app/service_inventory/ServiceList/index.tsx

```diff
@@ -193,48 +223,78 @@ export function ServiceList({ items, noItemsMessage }: Props) {
     : 'transactionsPerMinute';
 
   return (
-    <ManagedTable
-      columns={columns}
-      items={items}
-      noItemsMessage={noItemsMessage}
-      initialSortField={initialSortField}
-      initialSortDirection="desc"
-      initialPageSize={50}
-      sortFn={(itemsToSort, sortField, sortDirection) => {
-        // For healthStatus, sort items by healthStatus first, then by TPM
-        return sortField === 'healthStatus'
-          ? orderBy(
-              itemsToSort,
-              [
-                (item) => {
-                  return item.healthStatus
-                    ? SERVICE_HEALTH_STATUS_ORDER.indexOf(item.healthStatus)
-                    : -1;
-                },
-                (item) => item.transactionsPerMinute?.value ?? 0,
-              ],
-              [sortDirection, sortDirection]
-            )
-          : orderBy(
-              itemsToSort,
-              (item) => {
-                switch (sortField) {
-                  // Use `?? -1` here so `undefined` will appear after/before `0`.
-                  // In the table this will make the "N/A" items always at the
-                  // bottom/top.
-                  case 'avgResponseTime':
-                    return item.avgResponseTime?.value ?? -1;
-                  case 'transactionsPerMinute':
-                    return item.transactionsPerMinute?.value ?? -1;
-                  case 'transactionErrorRate':
-                    return item.transactionErrorRate?.value ?? -1;
-                  default:
-                    return item[sortField as keyof typeof item];
+    <EuiFlexGroup direction="column" responsive={false} alignItems="flexEnd">
+      <EuiFlexItem>
+        <EuiFlexGroup responsive={false} alignItems="center" gutterSize="xs">
+          <EuiFlexItem grow={false}>
+            <EuiToolTip
+              position="top"
+              content={i18n.translate(
+                'xpack.apm.servicesTable.tooltip.metricsExplanation',
+                {
+                  defaultMessage:
+                    'The service metrics are aggregated on transaction type "request", "page-load" or top transaction type',
```

Comment:
```suggestion
                    'The service metrics are aggregated on transaction type "request", "page-load", or the top transaction type available.',
```

@bmorelli25 mind taking a quick look at this? We're trying to inform the user that the inventory is now filtered by the default transaction types, unless a service does not support either, it will use the top transaction type found for that individual service.

Reply:
@formgeist, I really like your description to me of what the tooltip is trying to say. Based on that, what about something like this:

```suggestion
                    'Service metrics are aggregated on the default transaction types, "request" or "page-load". When neither is available, the service's top transaction type is used instead.',
```

Reply:
I can see how that might be too long though. If you think it is, I like your new suggestion, but with a few additional changes:

```suggestion
                    'Service metrics are aggregated on transaction type "request", "page-load", or the top available transaction type.',
```